indico/htdocs/js/indico/jquery/ajaxform.js

```diff
@@ -41,6 +41,13 @@
                 method: 'GET',
                 data: {}
             },
+            // only applicable if the form is not loaded with ajax.  setting it
+            // to true will assume that the form is not visible/editable at the
+            // beginning and thus not bind e.g. the onbeforeunload event.  when
+            // using this you most likely want to trigger `ajaxForm:externalShow`
+            // and `ajaxForm:externalHide` on the `context` element when the form
+            // becomes visible/accessible and/or hidden again
+            initiallyHidden: false,
```

Comment:
Couldn't find a better name for it. It basically tells the form that it's considered visible/active now (regarding onbeforeunload logic)